File: python/set02/problems/problem5.py
#takes an array as an argument and returns with only positive values
def positive(array):
    result = []
    for x in array:
        if x > 0:
            result.append(x)
    print(result);
arr = input("Enter an array with a space in between each number: ").split()
arr = [int(x) for x in arr]
positive(arr)

File: javascript/set06/problems/problem5.js
// Calculate the sum of numbers received in a comma delimited string

let x=prompt("Enter a string of numbers with a comman in between each: ");
let arr = x.split(",").map(Number);
let sum = 0
for (let i=0; i< arr.length ; i++){
    sum += arr[i];
}
console.log(sum);
